### DISCUSSION TITLE
Is Plex supported?

### DISCUSSION THREAD

**Discussion by bazmattaz** [jmbannon/ytdl-sub#915] [Q&A]
Hi folks, I've been trying to get shows to appear properly on Plex but with no luck. When I look through[ the guides](https://github.com/jmbannon/ytdl-sub/wiki/6.-Modifying-Your-Config-for-Your-Media-Player) it doesn't mention a setup for Plex. 

I'm basically looking for a guide on how to best setup a plex library to display YT content

**Comment by jmbannon** [comment]
https://ytdl-sub.readthedocs.io/en/latest/faq/index.html#plex-is-not-showing-my-tv-shows-correctly
